src/Tracing/GuzzleTracingMiddleware.php

```diff
@@ -29,9 +30,20 @@ public static function trace(?HubInterface $hub = null): Closure
                     return $handler($request, $options);
                 }
 
+                $partialUri = Uri::fromParts([
+                    'scheme' => $request->getUri()->getScheme(),
+                    'host' => $request->getUri()->getHost(),
+                    'port' => $request->getUri()->getPort(),
+                    'path' => $request->getUri()->getPath(),
+                ]);
```

Comment:
`l`: The only "worry" I have here is that a `MalformedUriException` is thrown for some reason. Although this might be totally unfounded since a URI was already there but maybe we want to try/catch anyway just to be 100% sure since not all PSR-7's might be created equal.

Reply:
Yeah, this might be a safer choice in the end. During my tests, the request normally fails with a cURL exception before this code runs.

Reply:
Yeah that's why I figured this might be overkill, so it might not be 100% needed. I also wouldn't know what to fallback too, because dropping the breadcrumb also seems wrong... so maybe leave as is and see what edge cases fall out. Don't think this could every happen outside synthetic tests anyway.